### DISCUSSION TITLE
Where can I report an issue I have or send feedback?

### DISCUSSION THREAD

**Discussion by danirabbit** [elementary/triage#621] [Q&A] [answered]
If I experience a problem or want to request a new feature, how can I report this to the development team?

**Comment by danirabbit** [comment, marked as answer]
elementary OS is developed 100% in the open on our [GitHub organization](https://github.com/elementary). If you're on elementary OS, we recommend using the built-in Feedback app to ensure your issue or feedback goes to the right place:

1. Open the Applications Menu and select the Feedback app
2. Select what you think is the closest match from the list provided
3. Follow the instructions in your web browser to file an issue
4. A detailed issue reporting guide [is available here](https://docs.elementary.io/contributor-guide/feedback/reporting-issues).

If you're not on elementary OS:

1. Visit our [GitHub organization](https://github.com/elementary) to view a list of all repositories
2. Use the “Find a repository…” search to search for the closest match for where you're seeing an issue
3. On a specific repository, select the “Issues” tab
4. Search the list for key words from your issue; try a few different ways of phrasing it
5. If you find your issue has already been reported, use the built-in 👍 reaction on the issue to mark it as affecting you, otherwise select the “New issue” button and follow the directions


Keep in mind that: 

- We may transfer your issue to the correct repository or mark it as a duplicate to keep the list of issues as organized as possible.
- It might take a while for someone to respond, remember that elementary OS is developed almost entirely by volunteers and there isn't a dedicated paid support staff

**Comment by doobes** [comment]
So,

How does this work if we have no idea which subsection the problem pertains to?

Asking users to figure out which subsection to post in seems sub-optimal, hence why they post in "Discussions".

**Reply by danirabbit** [reply]
This is covered in the linked contributor guide. Just do your best to pick a component. It's okay if you don't pick the right one because bug managers can move reports if need be.

Discussions can't be tracked and managed in the same way issue reports can. They are more like forum posts. Issues have important features that developers need to be able to properly track issues and make sure they are resolved

**Reply by AndreaMaietta** [reply]
I keep trying elementary but its crap like this that angers me.  I have once again downloaded elementary 8 and im having an issue with apps not pinning to dock or unpinning and there is no support. The answer above is not conducive to solutions and doesnt make people want to stay with elementary.  if i cant find a solution soon i will be unfortunately leaving this distro again

**Reply by danirabbit** [reply]
@AndreaMaietta if you've tried reporting an issue about this, I haven't seen it. But if you have, please link it so I can mark it as a duplicate. There is an open issue report for the dock already and myself and another developer as well as a QA engineer are actively investigating and we have proposed a couple solutions

**Reply by AndreaMaietta** [reply]
Will do it tomorrow am at work all day

**Reply by AndreaMaietta** [reply]
@danirabbit Where should I report it ?

**Reply by danirabbit** [reply]
@AndreaMaietta as I said, it's already been reported and I just merged in a fix a few minutes ago. We should be able to publish the update very soon

**Comment by AndreaMaietta** [comment]
Well thank you.  I appreciate the help makes me want to give elementary a longer run this time around
